build.gradle

```diff
@@ -3,9 +3,7 @@
 buildscript {
     repositories {
         google()
-        jcenter()
         mavenCentral()
-        maven { url 'https://jitpack.io' }
```

Comment:
Can not remove with repository, because  [LightCompressor](https://github.com/AbedElazizShe/LightCompressor) is used

Reply:
`LightCompressor` is used as a dependency library.

This is `buildscript` section which is not affecting generic dependencies but [plugins only](https://docs.gradle.org/current/userguide/declaring_repositories.html#sec:plugin-vs-build-repos).

Am I missing something?

Reply:
Oh, I see, you are right.